pkg/kn/commands/service/export.go

```diff
@@ -127,40 +127,59 @@ func NewServiceExportCommand(p *commands.KnParams) *cobra.Command {
 	}
 	flags := command.Flags()
 	commands.AddNamespaceFlags(flags, false)
+	flags.StringArray("include", nil, "Include certain info")
```

Comment:
I'd prefer to keep the `--with-*` format of option flags. In this case `--with-securit-context` or a short form`--with-sc`.